apps/microsoft-teams/frontend/src/components/config/ChannelSelectionModal/ChannelSelectionModal.tsx

```diff
@@ -66,6 +66,7 @@ const ChannelSelectionModal = (props: ChannelSelectionModalProps) => {
             <Radio
               id={channel.id}
               isChecked={selectedChannel.id === channel.id}
+              onChange={() => null /* noop, since clicking entire row selects this radio */}
```

Comment:
Adding this back 🤦  I removed this in an unrelated PR, because I thought it was an unnecessary prop that forma-36 was enforcing, but it is actually a browser/react warning that is pretty obnoxious in the console.  [conversation](https://github.com/contentful/apps/pull/6670#discussion_r1515232011)